### ISSUE TITLE
GroupedList and DetailsList: Level information is missing for screen readers

### ISSUE THREAD

**Issue by khmakoto** [microsoft/fluentui#15356]
The level information for grouped lists is missing for screen readers given that `aria-level` is only accepted by components with `row` groups when they are the children of components with `treegrid` roles and not `grid` roles. However, we cannot make this change currently since `treegrid` support by screen readers and browsers is generally in an incomplete state right now where all screen reader table navigation is thrown out and all the benefits of table structure would be effectively lost.

Right now there are no actionable items we can take on this bug but it is one that we should regularly circle back to in order to evaluate if there have been any advances in the browser + screen reader support for `treegrid` components.

**Comment by bcoard** [comment]
@khmakoto @smhigley  This appears to be resolved now.

**Comment by khmakoto** [comment]
I can confirm, I'll resolve this as `Resolution: Not an issue`.
